validphys2/src/validphys/results.py

```diff
@@ -949,6 +949,26 @@ def fits_groups_chi2_table(
     return res
 
 
+@table
+def fits_groups_nsigma_table(
+    fits_name_with_covmat_label, fits_groups, fits_groups_chi2_data, per_point_data: bool = True
+):
+    df = fits_groups_chi2_table(
+        fits_name_with_covmat_label, fits_groups, fits_groups_chi2_data, per_point_data
+    )
+    df = df.rename(columns={'ndata': 'ndata', '$\chi^2/ndata$': 'nsigma'})
+
+    for level_0 in df.columns.levels[0]:
+        ndata_col = (level_0, 'ndata')
+        nsigma_ndata_col = (level_0, 'nsigma')
+
+        # Apply the transformation and add the new column to the DataFrame
+        df[nsigma_ndata_col] = df.apply(
+            lambda row: (row[nsigma_ndata_col] - 1) / np.sqrt(2 / row[ndata_col]), axis=1
+        )
```

Comment:
I think this requires more comments or explanation but probably the docstring suffices